### PR TITLE
spotify current track genre finder (chosic.com)

### PR DESCRIPTION
**Pull request by quelhasu**
## Description

capture the current song played on spotify and search for its genre on chosic.com via favorite browser.

## Type of change

- [x] New script command

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/web-searches/chosic-genre-finder.applescript [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/osascript
+
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Chosic Genre Finder
```
Comment (unnamedd) [inline commands/web-searches/chosic-genre-finder.applescript:6]:
```suggestion
# @raycast.title Search Genre in Chosic
```

File: commands/web-searches/chosic-genre-finder.applescript [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# @raycast.mode inline
+
+# Optional parameters:
+# @raycast.packageName Spotify
```
Comment (unnamedd) [inline commands/web-searches/chosic-genre-finder.applescript:10]: If you are in the `web-searches` folder, makes more sense here to use the `Web Searches` in the `packageName`.
```suggestion
# @raycast.packageName Web Searches
```

File: commands/web-searches/chosic-genre-finder.applescript [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+# @raycast.icon https://cdn.iconscout.com/icon/free/png-512/spotify-11-432546.png
```
Comment (unnamedd) [inline commands/web-searches/chosic-genre-finder.applescript:11]: Here, the Spotify logo doesn't fit well. The best logo to use here is the Chosic logo, also, I'd like to ask you to submit the Chosic logo to the `image` folder under `web-searches`.

File: commands/web-searches/chosic-genre-finder.applescript [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+# Documentation:
+# @raycast.description Find the genre of the current Spotify track
```
Comment (unnamedd) [inline commands/web-searches/chosic-genre-finder.applescript:14]:
```suggestion
# @raycast.description Find the current Spotify track's gender in Chosic
```

File: commands/web-searches/chosic-genre-finder.applescript [hunk 5]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/osascript
```
Comment (unnamedd) [inline commands/web-searches/chosic-genre-finder.applescript:1]: Renaming the file to `search-gender-in-chosic.applescript` may fit better.

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hi @quelhasu, thanks for your first contribution in our repository.

Please, take a look into my comments and as soon you finish them, please, let me know for a second request.

It's really nice to have you here with us!

**Comment by quelhasu** [comment]
Thank you for your review, do I need to submit another PR?

**Comment by unnamedd** [comment]
> Thank you for your review, do I need to submit another PR?

Nope, as you already submitted your changes, I will review it again later on and let you know! Thanks a lot for your changes @quelhasu!

**Review by unnamedd — APPROVED** [review]
Thanks for the changes @quelhasu! 

Your PR is approved!

**Comment by quelhasu** [comment]
do I need to close this PR ?

**Comment by unnamedd** [comment]
> do I need to close this PR ?

No, you don't need. It will be merged as soon as possible. But, I need you to ask to rollback the addition of the new Script Command for Color Conversion. Would you please, create a second PR exclusive for that script command, please?

**Review by unnamedd — APPROVED** [review]
This PR is ready to be merged! 
@thomaspaulmann @PitNikola @fe9lix

**Comment by PitNikola** [comment]
Seems like there are some conflicts. @quelhasu, could you please rebase on latest master and then I'll merge it?

**Review by PitNikola — CHANGES_REQUESTED** [review]
Putting back to your queue for now

**Comment by quelhasu** [comment]
Are there other conflicts now?

**Comment by unnamedd** [comment]
Yes @quelhasu, you need to rebase `master` branch here and also need to remove Script Commands that doesn't belong to this pull request. 

Here is the list of files that must be removed from your pull request to have it merged:
image

**Comment by unnamedd** [comment]
Hi @quelhasu, I'm afraid you've made a mistake here. You deleted some files that must not be deleted. I'm checking the `master` branch in your fork and I can't see the files. 

As you made a `force push`, would you mind to explain a bit what you did to have the need to use the `force push`? A simple merge from our `master` branch to yours is definitely enough without have to delete any existing file.

**Comment by quelhasu** [comment]
I totally messed up with git on my two computers, hence the use of `force push` by mistake.

**Review by unnamedd — APPROVED** [review]
It's all good @quelhasu , thanks for act on it so quickly! 
@PitNikola, we are good to go again!

**Comment by PitNikola** [comment]
Cheers!

**Pull request merged**